campaigns/002-iw/run.py

```diff
@@ -0,0 +1,76 @@
+
+import timetracker
+
+
+def main(x):
+
+    print('Starting campaign 002-iw')
```

Comment:
The idea was that a campaign (should be) 100% self written anyway, thus the creator controls the output. I am not sure what's the correct way to do it.